irispy/sji.py

```diff
@@ -513,29 +515,31 @@ def read_iris_sji_level2_fits(filenames, memmap=False):
         ophaseix = hdulist[1].data[:, hdulist[1].header["OPHASEIX"]]
         slit_pos_x = hdulist[1].data[:, hdulist[1].header["SLTPX1IX"]]
         slit_pos_y = hdulist[1].data[:, hdulist[1].header["SLTPX2IX"]]
-        extra_coords = [('TIME', 0, times), ("PZTX", 0, pztx), ("PZTY", 0, pzty),
-                        ("XCENIX", 0, xcenix), ("YCENIX", 0, ycenix),
-                        ("OBS_VRIX", 0, obs_vrix), ("OPHASEIX", 0, ophaseix),
-                        ("EXPOSURE TIME", 0, exposure_times),
-                        ("SLIT X POSITION", 0, slit_pos_x*u.pix),
-                        ("SLIT Y POSITION", 0, slit_pos_y*u.pix)]
+        extra_coords = [('time', 0, times), ("pztx", 0, pztx), ("pzty", 0, pzty),
+                        ("xcenix", 0, xcenix), ("ycenix", 0, ycenix),
+                        ("obs_vrix", 0, obs_vrix), ("ophaseix", 0, ophaseix),
+                        ("exposure time", 0, exposure_times),
+                        ("slit x position", 0, slit_pos_x * u.pix),
+                        ("slit y position", 0, slit_pos_y * u.pix)]
         # Extraction of meta for NDCube from fits file.
         startobs = hdulist[0].header.get('STARTOBS', None)
-        startobs = parse_time(startobs) if startobs else None
+        startobs = Time(startobs) if startobs else None
```

Comment:
Regarding the comment from the imports section, I think here is an example where using ```parse_time``` should give the same output with the advantage that if the preferred sunpy time format is changed in the future and represented by ```parse_time```, irispy does not need any changes.